.circleci/config.yml

```diff
@@ -24,16 +24,13 @@ jobs:
           name: Start Postgres
           command: brew services start postgresql
 
-      - run:
-          name: Install Sourcery
-          command: brew install sourcery
-
-      - run:
-          name: Run Bootstrap OSS
-          command: make bootstrap-oss
-
       - run:
           name: Run Swift tests
           command: make test-oss
           environment:
             SNAPSHOT_ARTIFACTS: $CIRCLE_ARTIFACTS
+
+      - save_cache:
```

Comment:
I believe you need a corresponding `restore_cache` step, check out what I did for parser-printer:

https://github.com/pointfreeco/swift-parser-printer/blob/master/.circleci/config.yml#L11-L14